NSS-Day-Cohort-53/kennels-siennas
# Employees May Assign Locations

#### Changes Made
1. Modified file `LocationDetail.js` to include `constructNewEmployeeLocation()` to build a new employeeLocation object to be sent to the database, `getCurrentUser()` to check if authenticated user is an employee, and `EmployeeRepository` to communicate with the database.
2. Modified file `LocationsList.js` to include `updateLocations()` to set the state of `locations` for proper location rendering.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cg-empLocAssign
    ```
2. Open a new Terminal tab (⌘T or Ctrl+Shift+T) and navigate to the server directory.
3. Test app functionality.
    > Sign into the web-app as an employee (note that when signing in as a user, the employee assignment will not appear)
    > Notice that the locations are now visible
    > Click on a location and use the drop down menu to add a user to the location
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Resolves #1 #18 #40
